src/components/Agreements/ViewAgreement/Sections/Finances.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import { Accordion, Badge, Button, Icon } from '@folio/stripes/components';
+
+import FinancesAgreementLines from './FinancesAgreementLines';
+
+export default class Finances extends React.Component {
+  static propTypes = {
+    financesAgreementLines: PropTypes.arrayOf(PropTypes.object),
+    fetchMoreEresources: PropTypes.func,
```

Comment:
Can you remove this from the propTypes? It doesn't look like you're using or passing it in.